src/ln/msgs.rs

```diff
@@ -488,6 +488,17 @@ pub enum HTLCFailChannelUpdate {
 	},
 }
 
+/// For events which result in both a RevokeAndACK and a CommitmentUpdate, by default they should
+/// be sent in the order they appear in the return value, however sometimes the order needs to be
```

Comment:
maybe precise an example in which case the order needs to be variable?

Reply:
Added.